### PR TITLE
feat(docs): comptime functions `slice`, `rawSlice`, `ascii` and `crc32`

### LINKED ISSUE #899: Document new comptime functions: `slice`, `rawSlice`, `ascii` and `crc32`
https://github.com/tact-lang/tact/pull/787
&
https://github.com/tact-lang/tact/issues/444
&
https://github.com/tact-lang/tact/issues/796

### PR DESCRIPTION
**Pull request by novusnota**
## Issue

Closes #899.

## Checklist

- [x] I have updated CHANGELOG.md
- [x] I have run the linter, formatter and spellchecker
- [x] I did not do unrelated and/or undiscussed refactorings

### CODE REVIEW

**Comment by novusnota** [comment]
@Gusarich please, take a look!

**Comment by novusnota** [comment]
@Gusarich please, take a look!

**Review by Gusarich — DISMISSED** [review]

**Comment by Gusarich** [comment]
Sorry for the delay!

**Comment by anton-trunov** [comment]
@jeshecdom does everything seem clear to you?

**Comment by anton-trunov** [comment]
@novusnota there is a merge conflict in this pr

**Review by anton-trunov — DISMISSED** [review]

**Comment by novusnota** [comment]
I see the failure with pnpm change — either we pin the specific (prev.) pnpm version or add another command to its CI.

UPD: Apparently, pnpm [were doing that for a while](https://github.com/pnpm/pnpm/issues/4658). Adding `pnpm setup` prior to any actions with it might help.

**Comment by anton-trunov** [comment]
> or add another command to its CI

could you please expand on this option?

**Comment by novusnota** [comment]
pnpm is aware of its issues since [Jul 2022](https://github.com/pnpm/pnpm/issues/5000)

UPD: Wait, Tact's CI was completely skipped this time, investigating
UPD2: Ok, this is annoying, but I'll find a simple solution. Just pinning the version of pnpm may be enough.

**Comment by novusnota** [comment]
The last commit worked out in terms of pnpm. But, for some reason, after sourcing `.bashrc` on macOS, the Node.js version downgraded to 20 from 22, and the compilation failed due to `.isSubsetOf` not being present on Set object (known thing)

UPD: Found it! Default Node installed on [macos-latest runner (macOS 14 Arm64 at the moment) ](https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md#language-and-runtime) is 20. This seems like a bug of macos-runner, but that means sourcing `.bashrc` as advised by pnpm is actually wrong in this case.

**Comment by novusnota** [comment]
We stand victorious :)

File: .github/workflows/tact.yml [hunk 1]
```diff
@@ -231,7 +231,14 @@ jobs:
       - name: (pnpm) Test creation of new Blueprint projects
         if: ${{ matrix.package-manager == 'pnpm' }}
         run: |
-          npm i -g pnpm
+          npm i -g pnpm@9.12.2
```
Comment (anton-trunov) [inline .github/workflows/tact.yml:237]: let's add a link or something to explain this change, because it looks pretty involved

**Review by anton-trunov — APPROVED** [review]

**Pull request merged**